styles/mainStyle.css

```diff
@@ -866,22 +1002,178 @@ header {
   /* ID Styling */
   #presentation h2 {
     color: rgb(255, 255, 255);
-    font-family: "Inter", sans-serif;
+    font-family: 'Inter', sans-serif;
     font-size: 32px;
     font-weight: 500;
   }
 
   #presentation p {
     color: rgba(244, 245, 247, 1);
   }
+
+  .header_name {
+    border: 1px solid #f55800;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100%;
+  }
+
+  .header_sof {
+    align-self: flex-start;
+  }
+
+  .co1 {
+    border: 1px solid #f55800;
+  }
+
+  .header_title {
+    position: absolute;
+    background: #3c3a39;
+    height: 92px;
+    display: flex;
+    align-items: center;
+    justify-self: center;
+    justify-content: space-between;
+  }
+
+  .port_name {
+    width: 50%;
+    height: 24px;
+    margin-left: 12px;
+    margin-top: 10px;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: flex-start;
+    color: #fff;
+  }
+
+  .port_nav {
+    display: flex;
+    padding-left: 20px;
+    margin-left: 16.67%;
+    justify-content: flex-end;
+    align-items: flex-end;
+  }
+
+  .contact_section {
+    display: flex;
+    align-items: flex-start;
+    justify-content: flex-start;
+    flex-direction: row;
+    margin-left: 135px;
+    padding-top: 130px;
+    height: 590px;
+  }
+
+  .contact_header {
+    font-weight: bold;
+    font-family: Inter, sans-serif;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    font-style: normal;
+    width: 455px;
+    height: 356px;
+    color: #262626;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-content: flex-start;
+    margin-top: 130px;
+  }
+
```

Comment:
- [ ] Please, consider the alignment of the `contact-header` class, to be closer to the Figma design: in the template, it is aligned left. You can fix it using `text-align: left;`

> Figma template
<img width="659" alt="Screen Shot 2022-01-22 at 16 06 53" src="https://user-images.githubusercontent.com/49064106/150638230-b003c4b1-e4f9-4122-ac28-ee7fd39b077f.png">